File: AgentApp/Backend/src/main/java/rent/app/model/CarFuelType.java
package rent.app.model;

import lombok.Data;

import javax.persistence.*;
import javax.validation.constraints.Max;
import javax.validation.constraints.Min;

@Data
@Entity
@Table(name="CarFuelType")
public class CarFuelType {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column
    private String fuelType;

    @Column
    private boolean isRemoved;

}

File: AgentApp/Frontend/rent-ng-app/src/app/model/carFuelType.ts
export class CarFuelType{
  id: number;
  isRemoved: boolean;
  fuelType: string;

  constructor(id: number, isRemoved: boolean, name: string){
    this.id = id;
    this.isRemoved = isRemoved;
    this.fuelType = name;
  }
}
